### PR TITLE
Create NIPHL number check your answers page

### PR DESCRIPTION
**Pull request by liandra-giacomine**

### CODE REVIEW

File: app/models/TraderProfile.scala
Comment (besscerule) [inline app/models/TraderProfile.scala:100]: why aren't we using build niphls for this (the other function in this file)

File: app/controllers/CyaMaintainProfileController.scala
Comment (besscerule) [inline app/controllers/CyaMaintainProfileController.scala]: if we use build niphls we don't need to build the trader profile here because the function already does this

Comment (liandra-giacomine) [reply app/controllers/CyaMaintainProfileController.scala]: It seems to end up complicating things rather than simplifying them

Comment (besscerule) [reply app/controllers/CyaMaintainProfileController.scala]: Fair enough - could you also remove the build niphls and tests if it no longer used?

File: app/models/TraderProfile.scala
Comment (liandra-giacomine) [reply app/models/TraderProfile.scala:100]: We don't need to retrieve the profile for the page load

File: app/controllers/CyaMaintainProfileController.scala
Comment (liandra-giacomine) [reply app/controllers/CyaMaintainProfileController.scala]: Good point, removed

File: app/controllers/RemoveNiphlController.scala
Comment (ChrisCookOC) [inline app/controllers/RemoveNiphlController.scala]: Can we just remove if it is unused now?

**Review by ChrisCookOC — DISMISSED** [review]

**Comment by Ewan-James-Donovan** [comment]
Would approve after merge conflicts fixed :)

File: app/controllers/RemoveNiphlController.scala
Comment (liandra-giacomine) [reply app/controllers/RemoveNiphlController.scala]: Good shout, done

**Review by ChrisCookOC — APPROVED** [review]

**Pull request merged**
